index.bs

```diff
@@ -1447,16 +1459,17 @@ The following operations can be invoked by the client in an authenticator sessio
 This operation must be invoked in an authenticator session which has no other operations in progress. It takes the following
 input parameters:
 
-- The caller's RP ID, as determined by the user agent and the client.
+- The caller's [=RP ID=], as optionally decared by the [=[RP]=] and vetted by the user agent and the client, or defaulted to the
+    caller's [=environment settings object/origin=]'s [=effective domain=], c.f., [[#createCredential]].
 - The [=hash of the serialized client data=], provided by the client.
 - The [=[RP]=]'s {{PublicKeyCredentialEntity}}.
-- The user account's {{PublicKeyCredentialEntity}}.
+- The user account's {{PublicKeyCredentialUserEntity}}.
 - The {{PublicKeyCredentialType}} and cryptographic parameters requested by the [=[RP]=], with the cryptographic algorithms
     normalized as per the procedure in [[WebCryptoAPI#algorithm-normalization-normalize-an-algorithm]].
-- A list of {{PublicKeyCredential}} objects provided by the [=[RP]=] with the intention that, if any of these are known to the
-    authenticator, it should not create a new credential.
-- Extension data created by the client based on the extensions requested by the [=[RP]=].
-- The |requireResidentKey| parameter of the |options|.{{MakeCredentialOptions/authenticatorSelection}} dictionary.  
+- An optional list of {{PublicKeyCredential}} objects provided by the [=[RP]=] with the intention that, if any of these are
```

Comment:
Well, in the specific case of this authenticatorMakeCredential parameter, the `MakeCredentialOptions.excludeList` is different from the `|excludeList|` ephemeral variable in the `[[Create]](options)` method alg (aka #[createCredential](https://w3c.github.io/webauthn/#createCredential)). The former is filtered into the latter in "present #[createCredential](https://w3c.github.io/webauthn/#createCredential) step 23".

Also, the [webauthn "authenticator model"](https://w3c.github.io/webauthn/#authenticator-model) is hand-wavy "non-normative", as is CTAP itself. [CTAP section 4](https://fidoalliance.org/specs/fido-v2.0-rd-20161004/fido-client-to-authenticator-protocol-v2.0-rd-20161004.html#authenticator-api) says
> Note that this API level is conceptual and does not represent actual APIs. The actual APIs will be provided by each implementing platform.

see also issues #176 and #410.

So, perhaps we could leave the above list of authenticatorMakeCredential input params largely as-is, where it defines the values passed rather than using CTAP-specific parameter names?

Once we have a publicly-released updated CTAP spec, we could update this parameter list to say "for example, in CTAP this is foo parameter" (just a thought).